Jenkinsfile

```diff
@@ -84,6 +84,7 @@ timestamps {
         robotest : {
           if (env.RUN_ROBOTEST == 'run') {
             withCredentials([
+                [$class: 'StringBinding', credentialsId: 'GET_GRAVITATIONAL_IO_APIKEY', variable: 'GET_GRAVITATIONAL_IO_APIKEY'],
```

Comment:
nit: should we migrate to functions instead of this rudimentary interface? It is not like using assembly vs high-level language but still:
```
withCredntials([
  string(credentialsId: 'GET_GRAVITATIONAL_IO_APIKEY', variable: 'GET_GRAVITATIONAL_IO_APIKEY')
  file(credentialsId:'ROBOTEST_LOG_GOOGLE_APPLICATION_CREDENTIALS', variable: 'GOOGLE_APPLICATION_CREDENTIALS'),
])
```

Reply:
I'd prefer not to change that in this PR -- it introduces entirely new changes unrelated to the revert -- which then makes tracking history unclear

If this wasn't a revert, I'd be glad to incorporate this improvement.